spec/lib/twingly/url_spec.rb

```diff
@@ -134,7 +136,7 @@ def valid_urls
     end
 
     context "when given ASCII input" do
-      let(:ascii_url) { "http://www.twingly.com/öあ".force_encoding("ASCII-8BIT") }
+      let(:ascii_url) { "http://www.twingly.com/öあ".dup.force_encoding("ASCII-8BIT") }
```

Comment:
This broke without the `.dup`, I think this is ok but feel free to suggest something if you know other ways to create the ascii-string.